plugins/sqlite3/setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import setup
+
+PLUGIN_NAME = "sqlite3"
```

Comment:
I think we should ship with this one? as sqlite3 needs nothing as we will ship with pandas, unless you think its better to keep it separate?

Reply:
sure - what folder should we put it in?